subo/util/cache.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+// GetCacheDir returns and creates the cache directory
+func GetCacheDir() (string, error) {
+	cachePath, err := os.UserCacheDir()
```

Comment:
@denopink this is sweet! We do however have an existing pattern of using $TMPDIR instead of the cache dir, so I think I'd prefer we stay consistent there... for example we could use `$TMPDIR/suborbital/cache`

See the `builder` package for examples where we use the tmpdir :)

Reply:
I've opened an issue on this: #143

Reply:
Cool, I swapped out  `os.UserCacheDir()` with `os.TempDir()`